mlflow/transformers.py

```diff
@@ -824,13 +830,36 @@ def is_gpu_available():
         is_gpu = False
     return is_gpu
 
+def _try_load_model_with_device(model_instance, pipeline_path, device, conf):
+    load_model_conf = {}
+    # Assume if torch_dtype was specified in the conf, then it must be with a pipeline for which it's compatible.
+    if _TORCH_DTYPE_KEY in conf:
+        load_model_conf[_TORCH_DTYPE_KEY] = conf[_TORCH_DTYPE_KEY]
+
+    try:
+        load_model_conf["device"] = device
+        model = model_instance.from_pretrained(pipeline_path, **load_model_conf)
+    except (ValueError, TypeError):
+        print('Could not specify device parameter for this pipeline type', flush=True)
```

Comment:
Can we change this print statement to a logger warning statement?

Reply:
Done!